hpke_test.go

```diff
@@ -476,14 +490,47 @@ func verifyParameters(tv testVector, ctx cipherContext) {
 	assertBytesEqual(tv.t, tv.suite, "Incorrect parameter 'exporterSecret'", tv.exporterSecret, ctx.exporterSecret)
 }
 
+func verifyPublicKeysEqual(tv testVector, pkX, pkY KEMPublicKey) {
```

Comment:
Better to define an `Equals()` method on `KEMPublicKey` and `KEMPrivateKey`?

Reply:
I opted to keep this as is for consistency with the other checks we do.